nilearn/version.py

```diff
@@ -1 +1,97 @@
+# *- encoding: utf-8 -*-
+"""
+nilearn version, required package versions, and utilities for checking
+"""
+# Author: Alexandre Abraham, Philippe Gervais, Ben Cipollini
```

Comment:
I reckon you and I are the only ones who touched nilearn/version.py so I assume the comment should reflect that.